### DISCUSSION TITLE
DuckDNS non riesco

### DISCUSSION THREAD

**Discussion by Olodumdomotico** [indomus/forum#213] [HUB: Home Assistant]
Situazione attuale: Home assistant in versione nabbo OS che occupa tutto il raspberry

Comunque ho creato dominio in DUCKDNS
Installato add-on relativo ed impostato il "true" il token ed il dominio in configurazione....... all'avvio il log tutto ok
Nel router abilitate le porte ed eseguito forwording in 443 ed anche in 8123 (doppia solo per curiosità)

Usando browser da cellulare facendo attenzione che l'indirizzo sia https, non si collega
Provato anche medesimo indirizzo https://dominio.duckdns.org:8123 non si collega (ovviamente "dominio" sostituito con quello reale)

Cosa manca???

**Comment by snakuzzo** [comment]
Ma scusa eh...fai il forward dalla 443 esterna alla 8123 interna e ti colleghi dall'esterno sulla 8123?

Se il port forwarding che hai impostato è corretto devi collegarti su https://tuodominio e non su https://tuodominio:8123

**Reply by Olodumdomotico** [reply]
Forse non mi sono spiegato bene.....non ho fatto dalla 443 alla 8123 ma ho aperto solo 2 porte singole.
Se uso il protocollo https mi indirizza direttamente alla 443 senza specificarlo , quindi alla fine dell'indirizzo non scrivo ":443"
Solo come prova personale ho aperto anche la 8123 e se uso questa, alla fine dell'indirizzo devo aggiungere ":8123"
Però al momento non funziona su nessuna delle 2 porte

**Reply by snakuzzo** [reply]
Prima di metterti a fare prove su più porte...scegli quella che vuoi esporre ed usa quella.

La descrizione del tuo problema non è chiara...il "non funziona" può voler dire tutto o niente.
Devi descrivere bene il comportamento.

Dalla tua LAN via browser funziona?
Dall'esterno via browser funziona?
Dal cell stai usando l'app nativa o io browser?
Da cell stai provando da APN o dalla tua LAN via wifi?

**Reply by Olodumdomotico** [reply]
In effetti in discorso delle 2 porte è forviante, la cosa era nata per fare tentativi di funzionamento in realtà vorrei usare solo la 443

Attualmente da rete locale funziona tutto, ossia mi collego sia da PC che da cellulare su stessa wifi con APP
Invece se prendo il cellulare e spengo il wifi , lasciando il 3G e sul browser digito il dominio creato tramite duckdns mi dice "impossibile raggiungere il sito" e va in timeout .......come se non trova o non riesce ad accedere
Spero di aver spiegato meglio, se ti servono altri dettagli , volentieri

**Reply by 50494554524F** [reply]
con le connessioni mobili non puoi accedere dall'esterno... devi farti dare, se è un'opzione, un ip pubblico... o usare qualche server intermedio o il cloud di home assistant

**Reply by Olodumdomotico** [reply]
Ho guardato alcune guide in rete per capire eventuali errori, tutti si connettono da rete mobile tramite cellulare, ma non mi sembra che sia stato menzionato questo aspetto
Farò qualche ricerca in merito, grazie

**Reply by snakuzzo** [reply]
Scusa, ma che operatore hai ?

**Reply by 50494554524F** [reply]
aspè, fammi specificare meglio allora perché adesso mi hai messo il dubbio

tu puoi collegarti al tuo router anche da cellulare, è il tuo router che deve avere un ip pubblico...

**Reply by Olodumdomotico** [reply]
Operatore cellulare tiscali mobile, credo usa ponti tim
Operatore adsl fastweb

Più che ip pubblico, credo tu intenda ip fisso
Per ovviare al classico ip dinamico dovrebbe appunto bastare duckdns

**Reply by 50494554524F** [reply]
pubblico significa che il tuo pc può collegarsi alla "rete" e che la "rete/altri pc" possono collegarsi a te

tu hai un ip nattato o privato, cioè non accessibile dall'esterno... chiama fastweb e fatti dare un ip pubblico

**Reply by Olodumdomotico** [reply]
Ok mi informo grazie

**Reply by Olodumdomotico** [reply]
Riesco da router o altro a capire se sono sotto rete NAT del mio provider?

**Reply by Emilio-IT** [reply]
Ciao @Olodumdomotico,
a casa anch'io ho Fastweb come operatore per la connessione in fibra FTTC. Stessa cosa per un mio amico, lui però ha l'ADSL, come te.
In entrambi i casi eravamo partiti con la connessione "nattata". Immagino sia la norma e credo che anche per te sia così: la mia connessione presentava gli stessi sintomi che descrivi.
È stato sufficiente aprire un ticket con l'assistenza chiedendo l'IP statico e publico. Ora non ricordo quanto ci abbiano messo a risolverlo, comunque si parla di qualche ora, massimo un paio di giorni. Da lì in poi tutto fila liscio.
Buona fortuna!

**Reply by snakuzzo** [reply]
lavorano la richiesta nel giro di 24h di solito (senza disservizio ovviamente)

**Reply by Olodumdomotico** [reply]
Grazie
